packages/dds/tree/src/simple-tree/discrepancies.ts

```diff
@@ -263,27 +351,34 @@ function* getNodeDiscrepancies(
 }
 
 /**
- * The function to track the discrepancies between two field stored schemas.
+ * The function to track the discrepancies between a field view schema and a stored schema.
  *
  * @param keyOrRoot - If the key is missing, it indicates that this is the root field schema.
  */
 function* getFieldDiscrepancies(
-	view: TreeFieldStoredSchema,
+	view: FieldSchema,
 	stored: TreeFieldStoredSchema,
 	identifier: TreeNodeSchemaIdentifier | undefined,
 	fieldKey: FieldKey | undefined,
+	viewKindIsSequence = false,
 ): Iterable<FieldDiscrepancy> {
+	const normalizedView = normalizeFieldSchema(view);
+
 	// Only track the symmetric differences of two sets.
 	const findSetDiscrepancies = (
```

Comment:
this could use more documentation. I know it was undocumented before, but it would stil be nice to say what its return value is, since its hard to tell